src/Parser/DeathKnight/Frost/Modules/Features/WastedRimeProcs.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
```

Comment:
You can remove SpellLink since you never use it

Reply:
Removed it, can't you spellinks in tooltips.